include/dlr.h

```diff
@@ -222,8 +258,18 @@ int GetDLRWeightName(DLRModelHandle* handle, int index,
  message.
  */
 DLR_DLL
-int SetDLRInput(DLRModelHandle* handle, const char* name, const int64_t* shape,
-                const void* input, int dim);
+int SetDLRInput(DLRModelHandle* handle, const char* name, const int64_t* shape, const void* input,
+                int dim);
+
+/*!
+ * \brief Sets the input according the node name.
```

Comment:
Mention how this can only be used for TVM graph runtime models (such as those created by CreateTVMModel)

Reply:
Mention that these only work when `GetDLRBackend` returns tvm

Reply:
done